build/build.sh

```diff
@@ -9,10 +9,16 @@ if ! which go > /dev/null; then
 	exit 1
 fi
 
+GIT_COMMIT=$(git rev-parse HEAD)
+
+if [ $SOURCE_GIT_COMMIT ]; then
+	GIT_COMMIT=$SOURCE_GIT_COMMIT
+fi
```

Comment:
with `nounset` I believe this will error out where `SOURCE_GIT_COMMIT` isn't set. may i suggest the one-liner:
```suggestion
GIT_COMMIT=${SOURCE_GIT_COMMIT:-$(git rev-parse HEAD)}
```

Reply:
This suggestion feature is nifty. Thanks, Luke.

Reply:
Yes, my mistake. Thank you! @sosiouxme